### ISSUE TITLE
TypeError: undefined is not a function (on all javascript test)

### ISSUE THREAD

**Issue by andriansatria** [FreeCodeCamp/FreeCodeCamp#4848]
Challenge [Waypoint: Comment your JavaScript Code](http://www.freecodecamp.com/challenges/waypoint-comment-your-javascript-code) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 6.3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/37.0.2062.102 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript
//this as a comment

/* this is a test */
```

**Comment by sebemerson** [comment]
I think a few people have noticed the error.  Your code is correct, but the system will only accept it if you put this in:
```
// comment

/*/ comment /*/
```

It'll probably be corrected soon.

**Comment by AyonMKhan** [comment]
I am having this same issue. No matter what code I write, this message is displayed.

If you require a code sample that I ran:

function destroyer(arr) {
  
  return arr;
}

destroyer([1, 2, 3, 1, 2, 3], 2, 3);

I'm falling behind on my bonfires, I was hoping to start ziplines tomorrow.

Please fix this issue.

**Comment by Dajust** [comment]
It's a bug: a plugin is failing to initialize. I've posted the issue here :
https://github.com/FreeCodeCamp/FreeCodeCamp/issues/4853

**Comment by bugron** [comment]
Closing this issue. I believe this will be fixed in #4853.
